tardis/tests/tests_slow/report.py

```diff
@@ -144,6 +144,11 @@ def _generate_report(self, session):
             )
         )
         report_content += doc.unicode(indent=2)
+
+        # Quick hack for preventing log to be placed in narrow left out space
```

Comment:
As this line only affects the result part of the whole log I'd move it so it only works on `result` just after `result = ...`